tmt/utils.py

```diff
@@ -952,6 +952,30 @@ def remove_color(text):
     return re.sub(r'\x1B(?:[@-Z\\-_]|\[[0-?]*[ -/]*[@-~])', '', text)
 
 
+def validate_fmf_id(fmf_id):
```

Comment:
Seems it would be useful to have a function like this directly in `fmf` as well. @lukaszachy, what do you think?

Reply:
should I take some action?

Reply:
I am resolving this for now. we discussed with @lukaszachy and I will open a PR to FMF, to include it there, and then we can deprecate the function in this codebase.

Reply:
draft is here: https://github.com/psss/fmf/pull/112